std_msgs/String name
---
# Result code defintions
uint8 RESULT_SUCCESS=0
uint8 RESULT_NO_MAP_RECEIEVD=1
uint8 RESULT_UNDEFINED_FAILURE=255

uint8 result
